packages/connectivity_plus/connectivity_plus/example/integration_test/connectivity_plus_test.dart

```diff
@@ -25,11 +33,11 @@ void main() {
       expect(result, isNotNull);
     });
 
-    testWidgets('connectivity on Android emulator should be wifi',
+    testWidgets('connectivity on ${getPlatformName()} should be wifi',
         (WidgetTester tester) async {
       final result = await _connectivity.checkConnectivity();
 
       expect(result, ConnectivityResult.wifi);
-    }, skip: !Platform.isAndroid);
+    }, skip: !Platform.isMacOS && !Platform.isAndroid);
```

Comment:
As we saw on the test run results on GitHub Actions, MacOS expects ConnectivityResult ethernet.

Reply:
hmm weird. i pre-ran these tests on my fork and it worked just fine. any idea?

workflow job in a pr on my fork: https://github.com/werainkhatri/plus_plugins/actions/runs/3164967700/jobs/5153647025

Reply:
well, on my side the check seems to have passed. could you take another look?

Reply:
Maybe your laptop was connected to Wi-Fi but the GitHub Action images run on emulated Ethernet :)

Reply:
by "on my side", i meant the check on github states that it succeeded.

workflow job on this pr: https://github.com/fluttercommunity/plus_plugins/actions/runs/3165195385/jobs/5154030407